addon/components/sequential-render.js

```diff
@@ -242,11 +242,22 @@ export default Component.extend({
 
   fetchData: task(function* () {
     let { queryParams, taskOptions } = getProperties(this, 'queryParams', 'taskOptions');
-    let content = yield get(this, 'fetchDataTask').perform(queryParams, taskOptions);
+    let content;
+    if (get(this, 'getData')) {
```

Comment:
Can we use conditional operator to get the promise and then yield the variable? Also thoughts on adding a catch block to prevent abrupt breaks in the queue?

Reply:
Have updated in the latest commit